models/discordactions.js

```diff
@@ -549,6 +555,7 @@ const updateIdle7dUsersOnDiscord = async () => {
   let allUsersHavingGroupIdle7d = [];
   let groupIdle7dRole;
   let groupIdle7dRoleId;
+  const allMavens = [];
```

Comment:
can't we use from line 327 or make it global

Reply:
the thing is, these APIs are called in parallel, so i am not sure how i would do that? can you please help?